pkg/ddevapp/global_dotddev_assets/commands/host/dbeaver

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Run DBeaver against current db
+## Usage: dbeaver
+## Example: "ddev dbeaver [db] [user]"
+## HostBinaryExists: /mnt/c/Program Files/dbeaver/dbeaver.exe,C:\Program Files/dbeaver\dbeaver.exe,/Applications/dbeaver.app,/snap/bin/dbeaver-ce,/snap/bin/dbeaver-ee,/snap/bin/dbeaver
```

Comment:
I'm not sure whether `C:\Program Files/dbeaver\dbeaver.exe` is going to work here. 

```suggestion
## HostBinaryExists: /mnt/c/Program Files/dbeaver/dbeaver.exe,C:\Program Files\dbeaver\dbeaver.exe,/Applications/dbeaver.app,/snap/bin/dbeaver-ce,/snap/bin/dbeaver-ee,/snap/bin/dbeaver
```

Reply:
I fixed this